sdk/PrebidServerAdapter/PrebidServerModule/PBServerFetcher.m

```diff
@@ -47,17 +47,20 @@ - (void)makeBidRequest:(NSURLRequest *)request withCompletionHandler:(void (^)(N
         self.requestTIDs = [[NSMutableArray alloc] init];
     }
     @synchronized(self.requestTIDs) {
-        [self.requestTIDs addObject:params[@"tid"]];
+        if(params[@"tid"] != nil){
+            [self.requestTIDs addObject:params[@"tid"]];
```

Comment:
the tid logic can be removed from the fetcher now. since this field does not exist  on ortb.

Reply:
this method is removed... we dont use this now

Reply:
I found out that tid is required for legacy bidders, add it to the top request level as following: 
{
 "source": {
    "tid": "123"
  }
}

Reply:
done